core/src/main/java/org/apache/calcite/materialize/SqlStatisticProvider.java

```diff
@@ -16,16 +16,41 @@
  */
 package org.apache.calcite.materialize;
 
+import org.apache.calcite.plan.RelOptTable;
+
 import java.util.List;
 
 /**
  * Estimates row counts for tables and columns.
  *
```

Comment:
The comments are not that accurate, we should add isKey and isForeignKey

Reply:
I agree. Done.